lib/vsc/utils/fancylogger.py

```diff
@@ -443,15 +450,39 @@ def logToScreen(enable=True, handler=None, name=None, stdout=False):
 
     by default, logToScreen will log to stderr; logging to stdout instead can be done
     by setting the 'stdout' parameter to True
+
+    The `color` parameter enables or disables log colorization using
+    ANSI terminal escape sequences, according to the following values:
+
+    * when `color` is ``auto`` or ``None`` (default), then try to
+      auto-detect whether the output stream is connected to a terminal;
+    * when `color` is ``True`` or the string ``'yes'``, then turn on
+      log colorization unconditionally,
+    * any other value turns off log colorization unconditionally.
     """
     handleropts = {'stdout': stdout}
 
+    use_color_formatter = False  # default
+    if HAVE_COLOREDLOGS_MODULE:
+        if color is None or color == 'auto':
```

Comment:
the default shouldn't do anything, colored modules available or not. the current default means that as soon as the modules are present, the stdout logs are colorised.